__tests__/article-test.js

```diff
@@ -43,3 +43,33 @@ describe.only('Get article by ID', () => {
     });
 });
 
+describe('Get articles', () => {
+    it('Returns correct number of articles', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({body}) => {
+                expect(body.articles).toHaveLength(5);
```

Comment:
We could write this line in the next test and delete this one, feel free to group assertions on a integration test 🙂